docs/docsite/rst/guide_attributes.rst

```diff
@@ -233,10 +233,28 @@ As raw bytes cannot be expressed in YAML, to set an attribute with a byte array
 
 The value specified here is the bytes encoded as a base64 string.
 
+The :ref:`ansible.builtin.b64encode filter <ansible_collections.ansible.builtin.b64encode_filter>` can be used to encode strings on the fly, and the :ref:`ansible.builtin.file lookup <ansible_collections.ansible.builtin.file_lookup>` could be used to read data from a file.
+
+
+.. code-block:: yaml
+
+    - vars:
+        sig_data: "{{ lookup('ansible.builtin.file', '/path/to/my/sig') }}"
```

Comment:
Does this work for binary files, I'm never sure how bytes work when piped in filters. I assume the `file` lookup would try and return a text string rather than a byte string.

Reply:
Yeah I looked into for the very same question, `file` does convert to text, so I tried not to imply this would be good for bytes, I meant to show it as a convenient way to retrieve a string that might already be in a file.

Reply:
Updates to the wording, or removing the `file` example would be fine. I do think it's at least nice to show use of `b64encode` though

Reply:
It's just that using the `bytes` type implies the value is not valid text so reading from a file for the `b64encode` filter may or may not work. Maybe a `slurp` example is better?